usagov/vote-gov-drupal
# VOTE-1982 add view alter file for disabling caching for basics_modules

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-1982

## Description

Disable caching for taxonomy terms

## Deployment and testing

### Post-deploy steps
1. lando retune
2. Disable cache locally. In settings.local.php, comment out lines 35-119.
3. lando drush cr

### QA/Testing instructions

1. Visit all voter guide pages and confirm all headings are the level we expect (h2 versus h3).

## Checklist for the Developer




## Checklist for the Peer Reviewers
